apps/api/src/domain/games/memory-match/repositories/arango-memory-match-repository.ts

```diff
@@ -86,6 +86,10 @@ export class ArangoMemoryMatchRepository implements IMemoryMatchRepository {
         }
     }
 
+    async unpublish(_roundId: AggregateId): Promise<Error | AggregateId> {
+        throw new Error('Method not implemented.');
```

Comment:
Add a test case for this in the repository test. It should be really similar to the publish case.

Reply:
You've done this now.